net-proxy/squidguard/squidguard-1.5_beta-r2.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+WANT_AUTOMAKE=none
+
+inherit autotools user
+
+MY_P="squidGuard-${PV/_/-}"
+
+DESCRIPTION="Combined filter, redirector and access controller plugin for Squid"
+HOMEPAGE="http://www.squidguard.org"
+SRC_URI="http://www.squidguard.org/Downloads/Devel/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~sparc ~x86"
+
+IUSE="ldap"
+
+RDEPEND=">=sys-libs/db-2:*
```

Comment:
I'm quite sure you're linking to the library here, so `:*` is wrong. It should be `:=`.

Plus, since db:5.3 is stable already, you really want to either quick-stabilize this *or* fix the dependency to be <db-5 in the stable (+ revbump it while keeping stable keywords, i.e. `git mv`).